pkg/templates/openapi/ext.go

```diff
@@ -0,0 +1,315 @@
+package openapi
+
+import (
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+const OpenAPIVersion = "3.0.3"
```

Comment:
3.0.x is more widely used because most of the tools have not yet released stable support for version 3.1.0. Swagger only started supporting version 3.1 this year, and the OpenAPI Generator has not yet released stable support for 3.1.0.